### DISCUSSION TITLE
Testamonials

### DISCUSSION THREAD

**Discussion by alvarlagerlof** [alvarlagerlof/neurodiversity#340] [Ideas]
Allow collecting testimonials and life experiences to show for people to be able to sympathize easier

**Comment by pedroapfilho** [comment]
We could have a folder for the `content` that has `testimonials`, with `tags` on the head, so we could import it into the homepage and on specific pages

**Reply by alvarlagerlof** [reply]
Yup, it should be quite easy. I think the harder part is getting good testamonials.
